js/manager.js

```diff
@@ -407,6 +429,22 @@ const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
     }
   },
 
+  registerAmazonSlot(slotID, sizes, adUnit) {
```

Comment:
I wonder if we should move/put most of these new code/logic into a separated js, example: amazonManager.js, this way the code would be easier to maintain/extend in the short/long term, example:
amazonManager.js
 + registerSlot()  (registerAmazonSlot())
 + getSlots()  (getAmazonSlots())
 + count()  (isAmazonSlotsEmpty()).

what do you think guys?

Reply:
Absolutely agree, will move to a separate module